streams/src/test/java/org/apache/kafka/streams/integration/AbstractResetIntegrationTest.java

```diff
@@ -190,6 +181,10 @@ void cleanupTest() throws Exception {
             streams.close(Duration.ofSeconds(30));
         }
         IntegrationTestUtils.purgeLocalStreamsState(streamsConfig);
+        if (adminClient != null) {
+            adminClient.close(Duration.ofSeconds(10));
```

Comment:
Utils.closeQuiety? Same for streams.close above so that if one close fails it doesn't block the proper closing of other resources.

Reply:
Good idea, done